Fix metabug in utils.hpp

```
/builds/zrc-von-neumann-lab/spatial-computing/graphblas/include/graphblas/utils.hpp: In function 'bool grb::utils::equals(const T&, const T&, U, typename std::enable_if<(std::is_floating_point<_Tp>::value && std::is_integral<U>::value)>::type*)':
/builds/zrc-von-neumann-lab/spatial-computing/graphblas/include/graphblas/utils.hpp:155:9: error: 'cout' is not a member of 'std'
  155 |    std::cout << "Comparing " << a << " to " << b << " with relative tolerance "
      |         ^~~~
```

(Urgent as it is currently present in `develop`)

--- a/include/graphblas/utils.hpp
+++ b/include/graphblas/utils.hpp
@@ -36,6 +36,10 @@
 #include <graphblas/descriptors.hpp>
 #include <graphblas/utils/iscomplex.hpp>
 
+#ifdef _DEBUG
+ #include <iostream>
+#endif
+
 
 namespace grb {
 

